File: vortexdbg-api/src/main/java/com/vortexdbg/debugger/gdb/LastSignalCommand.java
package com.vortexdbg.debugger.gdb;

import com.vortexdbg.Emulator;

class LastSignalCommand implements GdbStubCommand {

    @Override
    public boolean processCommand(Emulator<?> emulator, GdbStub stub, String command) {
        stub.makePacketAndSend("S" + GdbStub.SIGTRAP);
        return true;
    }

}

File: vortexdbg-api/src/main/kotlin/com/vortexdbg/debugger/gdb/LastSignalCommand.kt
package com.vortexdbg.debugger.gdb

import com.vortexdbg.Emulator

internal class LastSignalCommand : GdbStubCommand {

    override fun processCommand(emulator: Emulator<*>, stub: GdbStub, command: String): Boolean {
        stub.makePacketAndSend("S" + GdbStub.SIGTRAP)
        return true
    }

}
